ivre/db/__init__.py

```diff
@@ -260,6 +260,12 @@ def __init__(self):
                                     help='show only results from this source')
         self.argparser.add_argument('--version', metavar="VERSION", type=int)
         self.argparser.add_argument('--timeago', metavar='SECONDS', type=int)
+        self.argparser.add_argument('--id', metavar='ID[,ID[...]]',
```

Comment:
Use `nargs='+'`, which does exactly what you want. This will be `argparse`-specific, so you will need to define a constant `USING_ARGPARSE` (such a constant is already used within some other tools in `bin/`).

Installations lacking `argparse` (that is, those using Python < 2.7 without the external `argparse` module installed) will only be able to select one ID at a time (which is not really a problem, particularly since `optparse` support is to be dropped soon).